airbyte-integrations/connectors/source-facebook-marketing/unit_tests/integration/response_builder.py

```diff
@@ -31,3 +34,19 @@ def error_reduce_amount_of_data_response() -> HttpResponse:
         "error": {"code": 1, "message": "Please reduce the amount of data you're asking for, then retry your request"},
     }
     return build_response(body=response, status_code=HTTPStatus.INTERNAL_SERVER_ERROR)
+
+
+def get_ads_response() -> HttpResponse:
```

Comment:
Should `create_response_builder` be used here?

Reply:
updated